README.rst

```diff
@@ -61,6 +61,33 @@ If we decided to use docker, we would need to run the image and publish the jupy
 Finally, we would need to copy the url shown in the docker to our browser.
 
 
+Supported protocols
+###################
+
+SyMPC supports the following protocols.
+
+.. table::
+
+    +--------------+------------------------+-----------------------+
+    | **Protocol** | **Security Guarantee** | **Number of Parties** |
+    +==============+========================+=======================+
+    | ABY3         | Semi-honest            |                     3 |
+    +--------------+------------------------+-----------------------+
+    | ABY3         | Malicious              |                     3 |
+    +--------------+------------------------+-----------------------+
+    | Falcon       | Semi-honest            |                     3 |
+    +--------------+------------------------+-----------------------+
+    | Falcon       | Malicious              |                     3 |
+    +--------------+------------------------+-----------------------+
+    | FSS          | Semi-honest            |                     2 |
+    +--------------+------------------------+-----------------------+
+    | SPDZ         | Semi-honest            |                    2+ |
+    +--------------+------------------------+-----------------------+
+
+It also support other types of technologies that we can see in
```

Comment:
It should be _" supported operations and Neural Network layers"_ not technology. You can name the separate file as "supported_ops"